release notes/v0.37.0.md

```diff
@@ -0,0 +1,73 @@
+
+## Enhancements and UX improvements
+
+- We continued work on the source map feature: add samples ([#2339](https://github.com/grafana/k6/pull/2339)) and improve loading and parsing ([#2355](https://github.com/grafana/k6/pull/2355)).
+- Updated installation from source instructions ([#2359](https://github.com/grafana/k6/pull/2359)). Thanks, @[karitham](https://github.com/Karitham), for making the change.
+- Introduced a Go-based thresholds parsing ([#2400](https://github.com/grafana/k6/pull/2400)).
+- Enabled more TC39 tests: [optional-chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) ([#2338](https://github.com/grafana/k6/pull/2338)) and async ([#2396](https://github.com/grafana/k6/pull/2396)).
+- We updated our [Code of Conduct](https://github.com/grafana/k6/blob/master/CODE_OF_CONDUCT.md) and actualized few GitHub URLs ([#2416](https://github.com/grafana/k6/pull/2416)).
+
+## Bugs fixed!
+
+- [`http.head`](https://k6.io/docs/javascript-api/k6-http/head-url-params/) started taking a body as a second argument ([#2402](https://github.com/grafana/k6/pull/2402)) by mistake in v0.36.0. It is now back to its old signature `http.head(url, [params])`.
+- Fixed [options.scenarios'](https://k6.io/docs/using-k6/options/#scenarios) JSON marshaling ([#2392](https://github.com/grafana/k6/pull/2392)).
+- Metrics' names display again in k6's Rest API ([#2421](https://github.com/grafana/k6/pull/2421)).
+- We improved misconfiguration detection for [`check`](https://k6.io/docs/javascript-api/k6/check-val-sets-tags)([#2387](https://github.com/grafana/k6/pull/2387)), [`http.batch`](https://k6.io/docs/javascript-api/k6-http/batch-requests)([#2415](https://github.com/grafana/k6/pull/2415)) and [metrics' constructors](https://k6.io/docs/javascript-api/k6-metrics/)([#2427](https://github.com/grafana/k6/pull/2427)).
+
+## Internals
+
+- We updated our CI to make a developer experience better. Dependency and linter checks run only for pull requests ([#2403](https://github.com/grafana/k6/pull/2403)).
+- This release also contains a bunch of the refactoring PRs that fix linters errors ([#2334](https://github.com/grafana/k6/pull/2334), [#2331](https://github.com/grafana/k6/pull/2331) and [#2341](https://github.com/grafana/k6/pull/2341)), remove global variables' usage ([#2336](https://github.com/grafana/k6/pull/2336), [#2358](https://github.com/grafana/k6/pull/2358), [#2353](https://github.com/grafana/k6/pull/2353) and [#2357](https://github.com/grafana/k6/pull/2357)) and remove unnecessary dependency ([#2313](https://github.com/grafana/k6/pull/2313)) which makes our codebase more consistent and maintainable.
+- The `headers`' parameter in k6's GRPC module marked as deprecated ([#2370](https://github.com/grafana/k6/pull/2370)).
+- Switched `envconfig` to a fork ([#2337](https://github.com/grafana/k6/pull/2337)) in order to abstract OS package and improve testability.
```

Comment:
It's good to emphasize it's not some random fork :)

```suggestion
- Switched `envconfig` to our own fork ([#2337](https://github.com/grafana/k6/pull/2337)) in order to abstract the `os` package and improve testability.
```

Reply:
`our own` it's @MStoykov 's fork :smile:

Reply:
Yeah, but it's an internal one we maintain, not from an external maintainer.

Reply:
True